lib/rownd_auth_cubit.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:async';
+import 'package:bloc/bloc.dart';
+import 'package:rownd_flutter_plugin/rownd.dart';
+import 'package:rownd_flutter_plugin/rownd_platform_interface.dart';
+import 'package:rownd_flutter_plugin/state/global_state.dart';
+
+enum AuthState { authenticated, unauthenticated, loading }
+
+class RowndAuthCubit extends Cubit<AuthState> {
```

Comment:
Yes, we can call it `RowndCubit`. I added the user data because I was trying to extract as much data from the RowndPlugin as possible, making it very similar to the `useRownd` hook in React. By providing the user data in that format, it allows users to customize their User class. Should it be its own cubit?

Reply:
Okay cool! Yup I think the user data should be in the cubit as well.